devdoc/refcount_requirements.md

```diff
@@ -0,0 +1,63 @@
+`refcount` requirements
+================
+
+## Overview
+
+`refcount` is a module that provides reference counting for a given structure.
+It wraps the structure that needs to be ref counted into another structre that contains an additional field (ref count).
```

Comment:
>structre [](start = 65, length = 8)

structure

Reply:
Fixed.

---
In reply to: [229806981](https://github.com/Azure/azure-c-shared-utility/pull/221#discussion_r229806981) [](ancestors = 229806981)